pkg/Microsoft.Private.PackageBaseline/Microsoft.Private.PackageBaseline.pkgproj

```diff
@@ -7,6 +7,9 @@
     <HarvestStablePackage>false</HarvestStablePackage>
     <SkipPackageFileCheck>true</SkipPackageFileCheck>
     <PackageIndexFileName>packageIndex.json</PackageIndexFileName>
+
+    <!-- Include symbols in package by default-->
+    <IncludeSymbolsInPackage Condition="'$(IncludeSymbolsInPackage)' == ''">true</IncludeSymbolsInPackage>
```

Comment:
NIT: For this one it's not required.  This is just a data package with some targets and a json file.